pom.xml

```
@@ -2,7 +2,7 @@
 <project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
     <modelVersion>4.0.0</modelVersion>
 
-    <groupId>no.statnett</groupId>
+    <groupId>io.statnett</groupId>
```

```suggestion
    <groupId>io.statnett.k3a</groupId>
```

While we are changing this.